samples/deleteSnippets.js

```diff
@@ -0,0 +1,158 @@
+// Copyright 2022 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+'use strict';
+
+async function main(
+  instanceId = 'YOUR_INSTANCE_ID',
+  tableId = 'YOUR_TABLE_ID',
+  deleteType = 'readRow'
+) {
+  // [START bigtable_deletes_print]
+  const {Bigtable} = require('@google-cloud/bigtable');
+  const bigtable = new Bigtable();
+
+  // TODO: We need to import this from mutation.ts, but I guess we need to compile typescript?
+  const DELETE_METHOD = 'delete';
```

Comment:
I'll revert this change as you suggested, but something I learned in my last job was to avoid something called magic string matching.

See `Avoid Magic Strings` in
https://javascript.plainenglish.io/javascript-best-practices-strings-booleans-and-constants-740b8f1e96dd

What I was actually hoping for here was to import `Mutation` and then use `Mutation.methods.DELETE` as per the TODO. It would be good to create enums for a lot of the repeated strings we use throughout the code just as enums are used in other languages to create programmatic links for cleaner code. Any thoughts?

Reply:
yes, I agree magic strings should be avoided! However, this snippet would be shown as a standalone example, so it might be confusing to see a variable replace just one string, and why I believe inlining it would make more sense for a sample. Let me know your thoughts.

Enums could be a nice improvement in the future for sure!

Reply:
Good point. I forgot about the aspect of this being a standalone example so I think you are 100% right here. Anyways, I reverted this change a couple hours ago so this should be good to go.